reference/5.1/Microsoft.PowerShell.Core/About/about_Types.ps1xml.md

```diff
@@ -1,65 +1,67 @@
+## Short description
+Explains how to use `Types.ps1xml` files to extend the types of objects that
+are used in PowerShell.
 
-## LONG DESCRIPTION
+## Long description
 
-Extended type data defines additional properties and methods ("members")
-of object types in PowerShell. There are two techniques for adding
-extended type data to a PowerShell session.
+Extended type data defines additional properties and methods ("members") of
+object types in PowerShell. There are two techniques for adding extended type
+data to a PowerShell session.
 
-- Types.ps1xml file: An XML file that defines extended type data.
-- `Update-TypeData`: A cmdlet that reloads Types.ps1xml files and defines
+- `Types.ps1xml` file: An XML file that defines extended type data.
+- `Update-TypeData`: A cmdlet that reloads `Types.ps1xml` files and defines
   extended data for types in the current session.
 
-This topic describes Types.ps1xml files. For more information about using the
+This topic describes `Types.ps1xml` files. For more information about using the
 `Update-TypeData` cmdlet to add dynamic extended type data to the current
 session see
 [Update-TypeData](../../Microsoft.PowerShell.Utility/Update-TypeData.md).
 
 ## About Extended Type Data
 
 Extended type data defines additional properties and methods ("members") of
-object types in PowerShell. You can extend any type that is supported
-by PowerShell and use the added properties and methods in the same way
-that you use the properties that are defined on the object types.
+object types in PowerShell. You can extend any type that is supported by
+PowerShell and use the added properties and methods in the same way that you
+use the properties that are defined on the object types.
 
-For example, PowerShell adds a `DateTime` property to all
-`System.DateTime` objects, such as the ones that the `Get-Date` cmdlet
-returns.
+For example, PowerShell adds a **DateTime** property to all `System.DateTime`
+objects, such as the ones that the `Get-Date` cmdlet returns.
 
 ```powershell
-PS C:> (Get-Date).DateTime
+(Get-Date).DateTime
+```
+
+```Output
 Sunday, January 29, 2012 9:43:57 AM
 ```
 
-You won't find the `DateTime` property in the description of the
-[`System.DateTime` structure](/dotnet/api/system.datetime),
-because PowerShell adds the property and it is visible only in Windows
-PowerShell.
+You won't find the **DateTime** property in the description of the
+[System.DateTime](/dotnet/api/system.datetime) structure, because PowerShell
+adds the property and it is visible only in PowerShell.
 
-To add the `DateTime` property to all PowerShell sessions, Windows
-PowerShell defines the `DateTime` property in the Types.ps1xml file in the
-PowerShell installation directory (`$PSHOME`).
+To add the **DateTime** property to all PowerShell sessions, PowerShell defines
+the **DateTime** property in the `Types.ps1xml` file in the PowerShell
+installation directory (`$PSHOME`).
 
 ## Adding Extended Type Data to PowerShell.
```

Comment:
This is pre-existing, but should this have a period at the end?  Also, it's sentence case (as are many other headers in this file).

Reply:
fixed